src/GslCore/AstAlgorithms.fs

```diff
@@ -26,6 +26,13 @@ type PrintState =
     insideAssembly: bool;
     quoteStrings: bool}
 
+/// Characters that appear in gene names that we need to escape with backticks
+/// when decompiling.
+let weirdGeneNameChars = [|'-'; ','; '('; ')'|]
```

Comment:
Check out gene YIL154C.   I think it has a ' in its name.  It's officially IMP2'   

https://www.yeastgenome.org/locus/S000001416

Reply:
Thanks, I'll add that character!  I thought briefly about allowing a broad range of special characters here but decided to keep it restricted to the known problem cases.